dynamodb/dynamo_lock_table.go

```diff
@@ -15,6 +16,26 @@ import (
 // running many automated tests in parallel, we use a counting semaphore
 var tableCreateDeleteSemaphore = NewCountingSemaphore(10)
 
+// Terraform requires the DynamoDB table to have a primary key with this name
+const ATTR_LOCK_ID = "LockID"
```

Comment:
`LockID` is the name of the column in DynamoDB. The value that gets written to it is the `key` parameter of the S3 bucket (perhaps with the S3 bucket name prefixed, I forget). Therefore, it's a unique lock for each unique state file.